src/docs/index.html

```diff
@@ -1,19 +1,85 @@
 <!doctype html>
 <html lang="en">
-	<head>
+	<head
+		prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#"
+	>
 		<title>Documentation | Caselaw Access Project</title>
 		<meta charset="utf-8" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<meta name="robots" content="noindex" />
-		<meta name="author" content="Your name" />
-		<meta name="description" content="Explore American Caselaw" />
-		<meta property="og:title" content="Caselaw Access Project" />
-		<meta property="og:description" content="Explore American Caselaw" />
-		<meta property="og:url" content="https://case.law/" />
+
+		<meta
+			property="og:title"
+			content="Documentation | Caselaw Access Project"
+		/>
+		<meta
+			property="twitter:title"
+			content="Documentation | Caselaw Access Project"
+		/>
+		<meta name="description" content="Caselaw Access Project Documentation" />
+		<meta
+			property="og:description"
+			content="Caselaw Access Project Documentation"
+		/>
+		<meta
+			name="twitter:description"
+			content="Caselaw Access Project Documentation"
+		/>
+		<meta name="twitter:card" content="summary_large_image" />
+		<meta
+			name="twitter:image"
+			content="https://case.law/images/og_image/documentation.png"
+		/>
+		<meta
+			property="og:image"
+			content="https://case.law/images/og_image/documentation.png"
+		/>
```

Comment:
FYI, these urls correspond to the filepath for the documentation opengraph preview image in the current CAP static directory, rather than the current CAP website.